### ISSUE TITLE
Explain use of intercept in ComponentwiseGradientBoostingSurvivalAnalysis

### ISSUE THREAD

**Issue by AshCK** [sebp/scikit-survival#68]
Hey Guys,

I have an issue with predict() in GradientBoostingSurvivalAnalysis.
As an example:
model=GradientBoostingSurvivalAnalysis(n_estimators=1000,random_state=0)
model.fit(x_train,y_train)   %xtrain(98,1400), y_train(98,2)
model.predict(x_test)   %x_test(45,1400)
 The output of predict() has 2D array(45,32) instead of (45,)!!

I don't have any idea why the output has the shape of (45,32)!!

**Comment by sebp** [comment]
Could you please provide a minimal working example to reproduce the problem?

**Comment by AshCK** [comment]
Sorry for the late Reply! 
I think the problem is related to my dataset because when I try with a dataset which is in package example it works fine.  

Thanks;)

**Comment by AshCK** [comment]
Hey Sebastian,
I hope you are doing well. I'm sorry if it is not relevant to this issue.
I try this code with example data:
data_x, y = load_veterans_lung_cancer()
x = encode_categorical(data_x)
x_train, x_test, y_train, y_test = train_test_split(x, y, test_size=0.4,  random_state= 0, shuffle=True)  
model=ComponentwiseGradientBoostingSurvivalAnalysis()
model.fit(x_train,y_train)
model.predit(x_test)
#to check the feature_impotance of features:
p=model.coef_
My quastion is that p should be the same size as feature (x_train.shape[1]=8)
but p size is 9. 
p=rray([ 0.        ,  0.        , -0.09387955,  0.20493769, -0.48064312,
       -0.01447176,  0.        ,  0.        ,  0.        ])

I also try with another dataset, and always the p size is one more than the feature size.
Thanks!

**Comment by sebp** [comment]
Unlike Cox's proportional hazards model, which has no intercept, ComponentwiseGradientBoostingSurvivalAnalysis is based on an accelerated failure time model, which has an intercept and is [implicitly accounted for](https://github.com/sebp/scikit-survival/blob/master/sksurv/ensemble/boosting.py#L194), i.e. `coef_[0]` is the intercept. I think we should clarify this in the documentation.
